searchapp/urls.py

```diff
@@ -5,6 +5,6 @@
 
 urlpatterns = [
 
-    url(r'^(?P<city>[a-z]+)/(?P<category>[a-z]+)$', views.search, name='search'),
```

Comment:
@spyshiv sir, 
Here as this is a search if i want the url to be displayed as 127.0.0.1:8000/search/delhi/bands
where delhi and bands are the city and category selected respectively.
What should I do in such case